mp/src/game/client/c_postprocesscontroller.cpp

```diff
@@ -0,0 +1,50 @@
+//====== Copyright � 1996-2008, Valve Corporation, All rights reserved. =======
+//
+// Purpose: stores map postprocess params
+//
+//=============================================================================
+#include "cbase.h"
+#include "c_postprocesscontroller.h"
+
+// memdbgon must be the last include file in a .cpp file!!!
+#include "tier0/memdbgon.h"
+
+#ifdef CPostProcessController
+#undef CPostProcessController
+#endif
```

Comment:
Since this is a client entity there is no need to do the ifdef stuff, just use the `C_` name.